pkg/build/strategies/dockerfile/dockerfile.go

```diff
@@ -424,18 +420,27 @@ func getDestination(config *api.Config) string {
 
 // getImageScriptsDir returns the directory containing the builder image scripts and a bool
 // indicating that the directory is expected to contain all s2i scripts
-func getImageScriptsDir(config *api.Config) (string, bool) {
+func getImageScriptsDir(config *api.Config, builder *Dockerfile) (string, map[string]bool) {
```

Comment:
The comment for this function does not accurately describe the function and needs to be updated.